### PR TITLE
Embargo Check uses Clinical Core Completion

### PR DESCRIPTION
**Pull request by joneubank**
A slight change is made to the check for if a file passes has sufficient clinical data to begin the embargo process. It is meant that a file can begin embargo if its associated donor has had all core clinical data submitted. In the donor data retrieved from the Clinical Service, this is identified by the property `completionStats.coreCompletionPercentage` . This service was previously checking the property `schemaMetadata.isValid` - this property has additional checks on top of the donor being core complete, since it requires that the donor has no migration errors vs the latest dictionary. We do not want the embargo process to be affected by migration errors.

So in summary, this change modifies the check for a file's clinical data to look for `completionStats.coreCompletionPercentage === 1`, now beginning embargo for any file whose donor is core complete, regardless of dictionary migration errors.

### CODE REVIEW

File: src/services/embargo.ts [hunk 1]
```diff
@@ -158,83 +168,88 @@ export function calculateEmbargoStage(dbFile: File): EmbargoStage {
...
-        `calculateEmbargoStartDate()`,
-        dbFile.fileId,
-        `file ${dbFile.fileId} has a donor with invalid clinical data and has no clinical exemption.`,
-      );
-      return undefined;
-    }
-  }
+	// Check for clinical exemption, if no exemption then we need valid clinical data
+	const clinicalExemption: boolean = dbFile.clinicalExemption !== undefined;
+	const clinicalCoreComplete: boolean = clinicalDonor?.completionStats?.coreCompletionPercentage === 1 || false;
```
Comment (joneubank) [inline src/services/embargo.ts:180]: This line is the only functional change.

Comment (ciaranschutte) [reply src/services/embargo.ts:180]: can just be:
```js
	const clinicalCoreComplete: boolean = clinicalDonor?.completionStats?.coreCompletionPercentage === 1;``` 
?

**Review by ciaranschutte — APPROVED** [review]

File: src/services/embargo.ts (hunk 1 above)
Comment (joneubank) [reply src/services/embargo.ts:180]: Yeup, guess I carried over the formatting from the previous version without thought. Will clean it up like this.
